legal-api/src/legal_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.58.0'  # pylint: disable=invalid-name
+__version__ = '2.58.1'  # pylint: disable=invalid-name
```

Comment:
we usually only bump version numbers before a release.  please revert.

Reply:
Okay. It's reverted.